docs/UserGuide.md

```diff
@@ -192,6 +194,16 @@ Format: `listo`
 Examples:
 * `listo`
 
+### Listing all orders: `incompleteo`
+
+Shows a list of all incomplete orders in ReadyBakey before and during a given date and time
+
+Format: `incompleteo DELIVERYDATETIME`
+
+Examples:
+* `incompleteo 25-12-2022 15:30`
```

Comment:
Explain what this returns?

Reply:
Got it, I'll explain how the format works